coordinator/internal/authority/userapi_test.go

```diff
@@ -95,9 +89,11 @@ func TestManifestSet(t *testing.T) {
 					[]byte("c"),
 				},
 			},
-			mSGetter: &stubManifestSetGetter{},
-			wantErr:  true,
+			wantErr: true,
 		},
+		// TODO(burgerdev): add test for dysfunctional history backend
+		// TODO(burgerdev): add test for workload owner keys in req	(present|absent|wrong)
+		// TODO(burgerdev): add test for last manifest (no owner keys)
```

Comment:
This TODO refers to the situation where a manifest with no workload owner keys is configured.